apps/fe-web/src/app/components/CallNow/CallNow.tsx

```diff
@@ -0,0 +1,18 @@
+import Image from 'next/image';
+import Phone from '../../assets/images/PhoneCall.svg';
+
+import styles from './CallNow.module.scss';
+
+const phoneNumber = 2195550114;
+const phoneDisplay = '(219) 555-0114';
```

Comment:
use https://www.npmjs.com/package/react-phone-number-input for number formatting